tests/acceptance/test_dbus.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+# Copyright 2020 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import os
+import pytest
+import time
+
+from common import put_no_sftp
+
+
+@pytest.mark.usefixtures("setup_board", "bitbake_path")
+class TestDBUS:
+    @pytest.mark.min_mender_version("2.6.0")
```

Comment:
These need to be `2.5.0`, as they refer to client version.

Reply:
As soon as we stabilize `mender-image-tests` I'm killing this thing.

Reply:
And well spotted :eyes:

Reply:
Fixed

Reply:
@oleorhagen 

> As soon as we stabilize `mender-image-tests` I'm killing this thing.

We cannot get the rid of this marker. This is the only way that we can run the tests when releasing bugfixes of old client (say, upcoming Mender 2.4.3 LTS release). That won't change with the `mender-image-tests` rework.